azurerm/data_source_app_service.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/terraform/helper/schema"
-	azSchema "github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
```

Comment:
Based on the files you changed in this PR due to package name change, keep using package alias might be a better idea.

Reply:
that's not needed in this instance as nothing else in these files is using a package named `azure` - we should only use aliases where necessary

Reply:
1. To avoid conflict with other packages.
2. Wrapping to ease future refactoring.